### PR TITLE
Portfolio Project-Redesign

### PR DESCRIPTION
**Pull request by kennedyowusu**
1. Build the portfolio page of the design using a mobile-first approach
2. Build the desktop version of the design
3. implement javascript functionalities in the design
4. Host the project of GitHub Pages
5. Install linter files and check for errors and fix them where necessary
6. Make the project responsive
7. Add popup dialogs to the see project buttons for the mobile version and generate the data dynamically using JavaScript objects and arrays
8. Add static data to the desktop popup dialog version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,134 +16,644 @@
...
+                  <li class="data-para-list-item">Ruby</li>
+                </ul>
+
+              </div>
+            </div>
+
+          </div>
+
+        </ul>
+      </div>
```
Comment (medaminedev66) [inline index.html:230]: - [ ] As requested in the project requirements, the projects section should be created dynamically with the information from the projects array (all the HTML of this section should be created when the page loads). Kindly remove the HTML code for the projects list and instead add them via JavaScript.

File: script.js [hunk 2]
```diff
@@ -4,4 +4,214 @@ const navLinks = document.querySelector('#nav-links');
...
+
+// Open and close popup For Desktop
+const deskTopPopup = document.getElementById('webPopup');
+
+// eslint-disable-next-line no-unused-vars
+function openDesktopPopup() {
+  deskTopPopup.classList.add('open-web-popup');
+}
+
+// eslint-disable-next-line no-unused-vars
```
Comment (medaminedev66) [inline script.js]: - [ ] Rather than disable the linter strive to fix them as this makes you more accustomed to the style guide of the tech.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @kennedyowusu  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional ✔️
- Git/GitHub flow best practices have been followed ✔️
- There are no linter errors ✔️
- You have a descriptive PR ✔️

## Required Changes ♻️
- [ ] Kindly ignore the `.history` folder by using gitignore. This makes it difficult for me to review your project.
- [ ] When I click on each project card, the pop-up does not show up. Kindly ensure the pop-up is displayed after clicking on each project card. The pop-up should display the info of the clicked project.
- [ ] Kindly use the [new Readme template](https://github.com/microverseinc/readme-template) that was provided by Microverse. This new readme template will provide more details about your project, making it stand out.
- [ ] Kindly stick to the Figma design as much as you can. Your pop-up desktop design should look exactly like the pop-up design provided in the Figma template:
  - The close button should be aligned to the right
  - The buttons should be below the text description
  - The technologies should be aligned to the left
  - You should use the same desktop font size provided if the Figma template.
  - You should the same margins, paddings, and gaps.
  - The content should fit the pop-up container.
  - The pop-up should be fixed and not scrolling.

Your design:
image

Figma design:
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @kennedyowusu , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has Linters errors, Kindly fix those Linters errors and ensure Linters result is green.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by kennedyowusu** [comment]
Thanks

**Pull request merged**